diesel_codegen/src/update.rs

```diff
@@ -138,14 +138,15 @@ fn save_changes_impl(
         let pk_field = pk.field_name.unwrap();
         quote_item!(cx,
             impl<'a> $struct_name {
-                $_pub fn save_changes<T>(&self, connection: &::diesel::Connection)
+                $_pub fn save_changes<T, Conn>(&self, connection: &Conn)
```

Comment:
This has the unfortunate side effect of causing `save_changes::<User>(&connection)` to turn into `save_changes::<User, _>(&connection)`. I'll fix this before the 0.5 release